test/end-to-end/fee_center/feeCenterCU.page.js

```diff
@@ -0,0 +1,126 @@
+
+/* jshint expr:true */
+/* global element, by, browser */
+
+/**
+ * This class is represents a fee center creation page in terms of structure and
+ * behaviour so it is a fee center creation page object
+ **/
+function CreateUpdateFeeCenterPage() {
```

Comment:
Many of these methods are already contained in helper libraries such as `FormUtils`.  Is there a reason to duplicate the effort here?